AntiCovid/app/src/main/res/navigation/nav_graph.xml

```diff
@@ -90,7 +88,11 @@
     <fragment
         android:id="@+id/profileFragment"
         android:name="com.swtug.anticovid.view.profile.ProfileFragment"
-        android:label="ProfileFragment" />
+        android:label="ProfileFragment" >
+        <action
+            android:id="@+id/action_profileFragment_to_loginFragment"
+            app:destination="@id/loginFragment" />
```

Comment:
Adding the popUpTo value didn't worked here ?

Reply:
Exactly